spec/shop_spec.rb

```diff
@@ -35,30 +33,32 @@
   end
 
   describe ".find" do
-    let(:shop) { Shop.new({name: 'RED', description: 'red apples'})
-    
-    it 'should return row with id 15 from the shops table' do
-      result = Shop.find(15)
-      
-      expect(result.name).to eq(shop.name)
-    end
-  end
-
-  describe "#update" do
     before { Shop.new({name: 'cafe', description: 'red apples'}).create }
 
     let(:last_shop) { Shop.all.last } #defining variable to use in spec
```

Comment:
you can get the opportunity and replace this part with the same as update `let(:shop_id) { Shop.new({name: 'Mandy', description: 'be careful'}).create }` as ` create` return the shop id created.

Reply:
Do you mean get rid of last_shop completely with something like this? 

    describe ".find" do
      let(:shop_id) { Shop.new({name: 'Mandy', description: 'be careful'}).create }

      it 'returns the given shop row from shops table' do
        shop = Shop.find(last_shop[:id])
        shop.name = 'Dandy'
        expect(shop.name).to eq('Dandy')